soufi/__init__.py

```diff
@@ -201,7 +244,22 @@ def main(
         elif distro == 'alpine':
             disc_source = func(name, version, aports_dir=aports)
         elif distro == 'centos':
-            disc_source = func(name, version, repos=repo)
+            optimal = 'optimal' in repo
```

Comment:
I don't understand what you're trying to do here. Why would 'optimal' be in the repo string?

Reply:
...because that's how you wrote it?  You offered `--repo optimal` as a shortcut in the CLI.

We *could* throw all that stuff down the stairs, but I was aiming to preserve existing functionality.

Reply:
Oh, drat, sorry, I completely forgot I had done that.